### DISCUSSION TITLE
Support for fractional market

### DISCUSSION THREAD

**Discussion by Oglaf** [ghostfolio/ghostfolio#1716] [Ideas]
Good day! I have recently started dipping my toes in this awesome project and noticed **A LOT** of error logs in portainer:

```
[Nest] 284  - 02/16/2023, 4:01:01 PM   ERROR [DataProviderService] Error: Could not get historical market data for WEGE3F.SA (YAHOO) from 2023-02-09 to 2023-02-16: [HTTPError] Not Found
https://query2.finance.yahoo.com/v7/finance/download/WEGE3F.SA?interval=1d&events=history&includeAdjustedClose=true&period1=1675900800&period2=1676505600
[Nest] 284  - 02/16/2023, 4:01:01 PM     LOG [DataGatheringProcessor (GATHER_HISTORICAL_MARKET_DATA)] Historical market data gathering has been completed for WEGE3F.SA (YAHOO).
```
The reason is because I have many fractional shares, in Brazil fractional shares are regulated by the stock market (not the brokerage) with their own symbols (they end with an extra 'F') and usually have a very small difference in price.

In Yahoo! Finance, profile information (sector, industry, country), news, analysis, historical data and dividend can only be found in the 'whole' stock.

In other words, the app should have a boolean to indicate if that activity is a fractional share and pull additional data from the "whole" share, unfortunately I'm not sure if Yahoo! Finance API has any relationship indicator between a fractional and whole.
